ui/dashboard/src/app/@routes/stats/stats.html

```diff
@@ -0,0 +1,17 @@
+<h2>Task states</h2>
+<div class="row">
+    <div class="col-6">
+        <app-chart-task-states [data]="taskStates" [chartStyle]="{ 'width.%': 100, 'height.%': 100, 'display': 'block' }">
```

Comment:
Is it recommended to put CSS directly in a HTML file?

Reply:
i fixed that